src/gtest/test_mempool.cpp

```diff
@@ -349,3 +350,62 @@ TEST(Mempool, SproutNegativeVersionTx) {
     boost::system::error_code ec;
     boost::filesystem::remove_all(pathTemp.string(), ec);
 }
+
+
+///////////////////////////////////////////////////////////////////////////////
+////////////////////////////// ProcessMempoolMsg //////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+class CNodeExt : public CNode
+{
+public:
+    CService ip(uint32_t i)
+    {
+        struct in_addr s;
+        s.s_addr = i;
+        return CService(CNetAddr(s), Params().GetDefaultPort());
+    }
+
+    CNodeExt():
+        CNode(INVALID_SOCKET, CAddress(ip(0xa0b0c002)), "", true)
+    {
+    }
+
+    std::set<CInv> pushedInvList;
+
+    void PushInvs(const char* pszCommand, const std::vector<CInv>& invVec)
```

Comment:
To avoid possible compilation warnings and to improve readability, I would add the `override` specifier to this method.

```suggestion
    void PushInvs(const char* pszCommand, const std::vector<CInv>& invVec) override
```